sqlglot/parser.py

```diff
@@ -1186,9 +1186,14 @@ def _parse_cte(self):
             alias=alias,
         )
 
-    def _parse_table_alias(self):
+    def _parse_table_alias(self, is_update=False):
```

Comment:
instead, you should pass in the table alias tokens

Reply:
that would be more general, makes sense 👍